Build-V3-Plugin-Monitoring-ios/Classes/TracesManagers/FirebasePerformanceTracesManager.swift

```diff
@@ -74,14 +77,44 @@ extension FirebasePerformanceTracesManager {
             }
         }
     }
+
+    func performance_didFinishTraceName(for index: Int) -> String {
```

Comment:
There is an issue here, 

the protocol method declare to return a optional String. (same for the others)